vividus-plugin-web-app/src/main/java/org/vividus/bdd/issue/CurrentPageUrlKnownIssueDataProvider.java

```diff
@@ -21,13 +21,15 @@
 import org.vividus.selenium.IWebDriverProvider;
 import org.vividus.softassert.issue.IKnownIssueDataProvider;
 
+import java.util.Optional;
+
 public class CurrentPageUrlKnownIssueDataProvider implements IKnownIssueDataProvider
 {
     @Inject private IWebDriverProvider webDriverProvider;
 
     @Override
-    public String getData()
+    public Optional<String> getData()
     {
-        return webDriverProvider.get().getCurrentUrl();
+        return Optional.ofNullable(webDriverProvider.get().getCurrentUrl());
```

Comment:
I believe it won't return `null`, it will throw `WebDriverException` which should be caught